### DISCUSSION TITLE
Cant connect to my Netbox after restart my Netbox Server because of Redis Update

### DISCUSSION THREAD

**Discussion by JLK20-11** [netbox-community/netbox#12517] [Help Wanted!]
Hello together,
I need help to get into my netbox since my restart... 
It seems like that redis done an update a few weeks ago and i didnt restarted my server since... now i did and it shows me that: 

Server Error

There was a problem with your request. Please contact an administrator.

The complete exception is provided below:

<class 'redis.exceptions.AuthenticationError'>

AUTH <password> called without any password configured for the default user. Are you sure your configuration is correct?

Python version: 3.10.6
NetBox version: 3.4.7

I googled the whole morning to find a way to login... but nothing helped :(.
I got an snapshot from the day i created netbox, but I filled it with so many informations that i dont want so set it back... hopefully anyone can help me with this :) 

Thanks and Redards,
Jan

**Comment by JLK20-11** [comment]
I did it by my own... 
Somehow redis deleted my default user out of the database and i had to create a new one with the old password set.
After that Netbox is working again

**Reply by JLK20-11** [reply]
but now i have to set anytime Ill reboot my server the default user in redis... i cant reboot without restarting the service and set the password, does anyone know how i can change it that it will stay in the default user? seems not right

**Reply by abhi1693** [reply]
It depends on how you have setup your Redis instance. If you are using docker container, make sure youhave mounted the relevant volume for it to persist the data. If you have used some other way, please provide more info so that others can help you out.
